diff -u orig/inputrc changed/inputrc
--- orig/inputrc	2016-02-13 00:18:16.828000000 +0100
+++ changed/inputrc	2016-02-13 00:49:21.000000000 +0100
@@ -38,8 +38,8 @@
 # "\e[6~": end-of-history
 
 # alternate mappings for "page up" and "page down" to search the history
-# "\e[5~": history-search-backward
-# "\e[6~": history-search-forward
+"\e[5~": history-search-backward
+"\e[6~": history-search-forward
 
 # mappings for Ctrl-left-arrow and Ctrl-right-arrow for word moving
 "\e[1;5C": forward-word
